src/app/pages/ArticlePage/Byline/index.test.tsx

```diff
@@ -0,0 +1,141 @@
+/* eslint-disable no-template-curly-in-string */
+import React, { ReactNode } from 'react';
+import { render, screen } from '@testing-library/react';
+import Byline from '.';
+import { ServiceContextProvider } from '../../../contexts/ServiceContext';
+import ArticleTimestamp from '../../../legacy/containers/ArticleTimestamp';
+import {
+  bylineWithNoRole,
+  bylineWithNoAuthor,
+  bylineWithNameAndRole,
+} from './fixture';
+
+interface Props {
+  fixture: any;
+  service: string;
+  children?: ReactNode;
```

Comment:
You can probably use the `PropsWithChildren` type here as well

Reply:
fixed 👍